tests/fixtures/requestSchema/dummySchema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Test Validation Schema",
+  "required": ["id"],
```

Comment:
We may add `description` here, to expose eventual vulnerability I've pointed in logic